lib/index.js

```diff
@@ -1,7 +1,7 @@
 var nn = require('bindings')('node_nanomsg.node');
 
 var util = require('util');
-var EventEmitter = require('events').EventEmitter;
```

Comment:
Let's keep this here, since we end up requiring it twice down below.  I _know_ requires are cached, but let's keep imports at the top of the file, and remove the two requires down below.